specs/DesignDiscussions/InputLabels.md

```diff
@@ -224,6 +308,9 @@ Either of these approaches would allow card authors who know their cards will be
 
 In the case of a host using the `labelFor` property there would be no negative impact on down-level platforms other than not providing the association. Because the TextBlock will still exist, only the association will be dropped when rendered on a platform that doesn't support this feature.
 
+> ### Conclusion 
+> It was decided that the approach to follow was the one presented in the Option 1: Providing the "label" property in all Input elements. isRequired hints will also only be rendered in label elements.
+
 ## Placeholders
```

Comment:
>Placeholder [](start = 3, length = 11)

This doesnt cover adding in PLaceholders for Input.ChoiceSet yes?

Reply:
Based on the information gathered from asking the Accessibility team we decided not to use labels as placeholders as that affects user experience and accessibility